File: packages/insomnia-httpsnippet/test/fixtures/output/ocaml/cohttp/application-form-encoded.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.add (Header.init ()) "content-type" "application/x-www-form-urlencoded" in
let body = Cohttp_lwt_body.of_string "foo=bar&hello=world" in

Client.call ~headers ~body `POST uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/application-form-encoded.js
var unirest = require('unirest');

var req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'application/x-www-form-urlencoded'
});

req.form({
  foo: 'bar',
  hello: 'world'
});

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
